### ISSUE TITLE
Chunk app/app.module does not have associated output files

### ISSUE THREAD

**Issue by ppc52776** [clbond/angular-ssr#50]
Hello, I can't run the ng-render. It outputs "Chunk app/app.module does not have associated output files".
Step to re-product:
```
ng new project
cd project
ng build
./node_modules/.bin/ng-render
```

angluar version:
```
@angular/cli: 1.1.3
node: 8.1.2
os: linux x64
@angular/animations: 4.2.4
@angular/common: 4.2.4
@angular/compiler: 4.2.4
@angular/core: 4.2.4
@angular/forms: 4.2.4
@angular/http: 4.2.4
@angular/platform-browser: 4.2.4
@angular/platform-browser-dynamic: 4.2.4
@angular/router: 4.2.4
@angular/service-worker: 1.0.0-beta.8
@angular/cli: 1.1.3
@angular/compiler-cli: 4.2.4
@angular/language-service: 4.2.4
```

Output:
[output.txt](https://github.com/clbond/angular-ssr/files/1100473/output.txt)
```
[info] Rendering application from source (working path: /tmp/18eb39ace1b68) 0l
Unhandled Promise rejection: Chunk app/app.module does not have associated output files ; Zone: <root> ; Task: Promise.then ; Value: { Error: Chunk app/app.module does not have associated output files                                                                      
    at CompilerException.Exception [as constructor] (/tmp/ng/node_modules/angular-ssr/source/exception.ts:6:5)
    at new CompilerException (/tmp/ng/node_modules/angular-ssr/build/exception.js:63:42)
    at WebpackModuleLoader.lazy (/tmp/ng/node_modules/angular-ssr/source/application/compiler/webpack/loader.ts:26:13)
    at WebpackModuleLoader.load (/tmp/ng/node_modules/angular-ssr/source/application/compiler/webpack/loader.ts:15:17)
    at /tmp/ng/node_modules/angular-ssr/source/application/builder/from-source.ts:37:22
    at ZoneDelegate.invoke (/tmp/ng/node_modules/zone.js/dist/zone-node.js:391:26)
    at Zone.run (/tmp/ng/node_modules/zone.js/dist/zone-node.js:141:43)
    at /tmp/ng/node_modules/zone.js/dist/zone-node.js:818:57
    at ZoneDelegate.invokeTask (/tmp/ng/node_modules/zone.js/dist/zone-node.js:424:31)
    at Zone.runTask (/tmp/ng/node_modules/zone.js/dist/zone-node.js:191:47)
    at drainMicroTaskQueue (/tmp/ng/node_modules/zone.js/dist/zone-node.js:584:35)
    at ZoneTask.invoke (/tmp/ng/node_modules/zone.js/dist/zone-node.js:490:25)
    at data.args.(anonymous function) (/tmp/ng/node_modules/zone.js/dist/zone-node.js:1274:25)
    at FSReqWrap.oncomplete (fs.js:135:15)
  innerException: undefined,
  __zone_symbol__currentTask: 
   ZoneTask {
     _zone: 
      Zone {
        _properties: {},
        _parent: null,
        _name: '<root>',
        _zoneDelegate: [Object] },
     runCount: 0,
     _zoneDelegates: null,
     _state: 'notScheduled',
     type: 'microTask',
     source: 'Promise.then',
     data: undefined,
     scheduleFn: undefined,
     cancelFn: null,
     callback: [Function],
     invoke: [Function] } } Error: Chunk app/app.module does not have associated output files
....
```

**Comment by JPM84** [comment]
Same issue as #47.

**Comment by clbond** [comment]
Duplicate of #47

**Comment by clbond** [comment]
This has been resolved in `0.10.19` btw
